gsoc/gsoc2016/ideas_yt.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title:  "Ideas for yt"
+show_main: false
+ideas_team: yt
+---
+
+### Integrate yt plots with interactive matplotlib backends
+
+*Suggested Mentor(s):* Nathan Goldbaum
```

Comment:
We require two mentors per project (per student actually), so it would be good if we have at least someone named as a backup mentor. Normally the backup mentor is the one that could help the student when the first one is away on holidays or in a conference - or if the mentor disappears.  If no one is able it would default to the sub-org admin (ie. @MatthewTurk)